conftest.py

```diff
@@ -7,14 +7,18 @@
 from django.test.utils import get_runner
 
 
-def run():
+def pytest_configure():
     os.environ["DJANGO_SETTINGS_MODULE"] = "tests.settings"
     django.setup()
+
+
+def run(path):
+    pytest_configure()
```

Comment:
@fsbraun
I noticed you did the same thing in `djangocms-text`.
If I place the content of `pytest_configure` into the `run` function,
I get an `ImproperlyConfigured` error but it works fine like so.
Do you know the reason?

Reply:
I’m afraid not. Maybe pytest does execute pytest_configure only?

Reply:
Figured it out. pytest hook before test module is run
Never mind!